homeassistant/helpers/config_validation.py

```diff
@@ -122,6 +122,25 @@ def boolean(value: Any) -> bool:
     raise vol.Invalid("invalid boolean value {}".format(value))
 
 
+def number(value: Any) -> Union[int, float]:
```

Comment:
Please move this to the module/package where it's used. We only add functions here that are used in many different integrations.

Reply:
Ok. Done